src/api/app/components/request_decision_component.rb

```diff
@@ -42,14 +42,14 @@ def show_add_submitter_as_maintainer_option?
   end
 
   def accept_with_options_allowed?
-    single_action_request && @is_target_maintainer && @bs_request.state.in?(%i[new review])
+    @is_target_maintainer && @bs_request.state.in?(%i[new review])
```

Comment:
This is the same as the `accept_request?` policy.

Reply:
Right

Reply:
Fixed